### PR TITLE
docs: front page

### PR DESCRIPTION
**Pull request by ArielElp**
### Description of the Changes

Please add a detailed description of the change, whether it's an enhancement or a bugfix.
If the PR is related to an open issue please link to it.

### Check List

- [x] Changes have been done against dev branch, and PR does not conflict
- [x] Docs have been updated
- [x] PR title is follow the convention: `<docs/feat/fix/chore>(optional scope): <description>`, e.g: `fix: minor typos in code`

---
This change is [Reviewable](https://reviewable.io/reviews/starkware-libs/starknet-docs/17)

### CODE REVIEW

File: src/config/home.json [hunk 1]
```diff
@@ -1,38 +1,32 @@
...
     "img": "undraw_docusaurus_mountain.svg",
     "title": "what is starkNet",
-    "text": "intrudaction to starknet: how it works, main concepts glossary, FAQs"
-  },
-  {
-    "linkName": "docs/CLI/commands",
-    "img": "undraw_docusaurus_tree.svg",
-    "title": "building on starknet",
-    "text": "useful link to help write and deploy your own contracts on starknet"
+    "text": "Introduction to StarkNet: how it works, main concepts glossary, FAQs"
```
Comment (bbrandtom) [inline src/config/home.json:6]: We don't have an intro to StarkNet yet. I would change this sentence to describe the technical docs. We don't have a glossary nor FAQs here.

File: src/config/home.json [hunk 2]
```diff
@@ -1,38 +1,32 @@
 [
   {
-    "linkName": "docs/CLI/commands",
+    "linkName": "docs/Intro",
     "img": "undraw_docusaurus_mountain.svg",
     "title": "what is starkNet",
```
Comment (bbrandtom) [inline src/config/home.json:5]:
```suggestion
    "title": "What is StarkNet",
```

File: src/config/home.json [hunk 3]
```diff
@@ -1,38 +1,32 @@
...
+    "text": "Introduction to StarkNet: how it works, main concepts glossary, FAQs"
   },
   {
-    "linkName": "docs/CLI/commands",
+    "linkName": "https://www.cairo-lang.org/docs/hello_starknet/index.html",
     "img": "undraw_docusaurus_react.svg",
-    "title": "dev tools",
-    "text": "tools for development and deployment"
+    "title": "Hello StarkNet!",
+    "text": "Your StarkNet first steps"
```
Comment (bbrandtom) [inline src/config/home.json:12]: maybe "Your first steps in StarkNet"

File: src/config/home.json [hunk 4]
```diff
@@ -1,38 +1,32 @@
...
-    "text": "a link to Voyager, starknet alph block explorer"
+    "text": "Voyager, StarkNet's block explorer"
   },
   {
-    "linkName": "docs/CLI/commands",
+    "linkName": "https://swagtimus.substack.com/",
     "img": "undraw_docusaurus_react.svg",
     "title": "latest updates",
-    "text": "stay up to date with all the latest blog posts, news, and events"
+    "text": "StarkNet newsletter"
```
Comment (bbrandtom) [inline src/config/home.json:30]:
```suggestion
    "text": "StarkNet's unofficial newsletter"
```

**Review by bbrandtom — CHANGES_REQUESTED** [review]
Tried using GitHub review - let me know how it works.

**Review by ArielElp — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/starkware-libs/starknet-docs/17)* status: 0 of 4 files reviewed, 4 unresolved discussions (waiting on @bbrandtom)

___
*[src/config/home.json, line 5 at r1](https://reviewable.io/reviews/starkware-libs/starknet-docs/17#-MxpSsVYTJjBp68hYRG6-r1-5:-MxpTOce-VcfhprofQk8:b-896fix) ([raw file](https://github.com/starkware-libs/starknet-docs/blob/223ea225f1bb912862b5e852bf2af984be1e409d/src/config/home.json#L5)):*
<details><summary><i>Previously, bbrandtom (Tom Brand) wrote…</i></summary><blockquote>

```suggestion
    "title": "What is StarkNet",
```
</blockquote></details>

Done.
___
*[src/config/home.json, line 6 at r1](https://reviewable.io/reviews/starkware-libs/starknet-docs/17#-MxpSsVYTJjBp68hYRG6-r1-6:-MxpUgtM5pAJhO5sjAl0:b-896fix) ([raw file](https://github.com/starkware-libs/starknet-docs/blob/223ea225f1bb912862b5e852bf2af984be1e409d/src/config/home.json#L6)):*
<details><summary><i>Previously, bbrandtom (Tom Brand) wrote…</i></summary><blockquote>

We don't have an intro to StarkNet yet. I would change this sentence to describe the technical docs. We don't have a glossary nor FAQs here.
</blockquote></details>

Done.
___
*[src/config/home.json, line 12 at r1](https://reviewable.io/reviews/starkware-libs/starknet-docs/17#-MxpSsVYTJjBp68hYRG6-r1-12:-MxpTwQc0epKRV5n8t4S:b-896fix) ([raw file](https://github.com/starkware-libs/starknet-docs/blob/223ea225f1bb912862b5e852bf2af984be1e409d/src/config/home.json#L12)):*
<details><summary><i>Previously, bbrandtom (Tom Brand) wrote…</i></summary><blockquote>

maybe "Your first steps in StarkNet"
</blockquote></details>

Done.
___
*[src/config/home.json, line 30 at r1](https://reviewable.io/reviews/starkware-libs/starknet-docs/17#-MxpSsVYTJjBp68hYRG6-r1-30:-MxpU3z90hYG5urPeYXi:b-896fix) ([raw file](https://github.com/starkware-libs/starknet-docs/blob/223ea225f1bb912862b5e852bf2af984be1e409d/src/config/home.json#L30)):*
<details><summary><i>Previously, bbrandtom (Tom Brand) wrote…</i></summary><blockquote>

```suggestion
    "text": "StarkNet's unofficial newsletter"
```
</blockquote></details>

Done.

**Review by bbrandtom — COMMENTED** [review]
:lgtm: 

> Reviewed 2 of 4 files at r1, 2 of 2 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/starkware-libs/starknet-docs/17#-:-Mxs5lxo7dpuCs3_wwA9:bg7igkv)* status: all files reviewed, 1 unresolved discussion (waiting on @bbrandtom)

**Review by ArielElp — COMMENTED** [review]
:lgtm:

> *[Reviewable](https://reviewable.io/reviews/starkware-libs/starknet-docs/17#-:-Mxx4QC496-4krGf5o9I:bnfp4nl)* status: all files reviewed, 1 unresolved discussion (waiting on @bbrandtom)

**Review by ArielElp — COMMENTED** [review]
> Dismissed @bbrandtom from a discussion.
> *[Reviewable](https://reviewable.io/reviews/starkware-libs/starknet-docs/17)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @ArielElp)

**Pull request merged**